eZ/Bundle/EzPublishRestBundle/Resources/config/services.yml

```diff
@@ -226,6 +226,15 @@ services:
         arguments:
             - "@ezpublish.api.service.search"
 
+    ezpublish_rest.controller.session:
+        class: eZ\Publish\Core\REST\Server\Controller\SessionController
+        parent: ezpublish_rest.controller.base
+        arguments:
+            - '@ezpublish_rest.session_authenticator'
+            - '%ezpublish_rest.csrf_token_intention%'
+            - '@?security.csrf.token_manager'
+            - '@?security.csrf.token_storage'
```

Comment:
I thought we standardised on double quotes for these...? (Very minor issue, it's just nice to be consistent.)